ctapipe/image/hillas.py

```diff
@@ -70,7 +96,11 @@ def hillas_parameters(geom, image):
     # The ddof=0 makes this comparable to the other methods,
     # but ddof=1 should be more correct, mostly affects small showers
     # on a percent level
-    cov = np.cov(delta_x, delta_y, aweights=image, ddof=0)
+    try:
+        cov = np.cov(delta_x, delta_y, aweights=image, ddof=0)
+    except ValueError:
```

Comment:
The ValueError will also be raised in other cases. E.g. when the weights really are negative. 

Do we really want to support masked arays here? What is the advantange?

Many things, including simple sums and so on, turned out much slower using masked arrays, so why bother?